package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "react-native-with-moxy",
+  "version": "0.0.1",
+  "description": "The https://{project-domain} web app",
+  "author": "MOXY <hello@moxy.studio>",
+  "private": true,
+  "license": "UNLICENSED",
+  "keywords": [
+    "app",
+    "native"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:{project-org}/{project-name}"
+  },
+  "scripts": {
+    "test": "jest --coverage",
```

Comment:
```suggestion
    "test": "jest",
```

Reply:
I understand this as it can go as an option, but how many times do you run tests without coverage? You almost always need to check for coverage in my opinion 😛

Reply:
I do run tests locally without checking coverage most of the times. The CI always checks coverage anyway, so we're safe in this regard. I'd rather force developers to pass `--coverage` than to pass `--no-coverage`. Whenever developers want to run tests and get a more detailed output, they should pass whatever flags they need.